api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -269,7 +271,7 @@ public long calculateDuration() {
             long elapsedTime = endTimeNS - getNodes().get(0).getBaseTime();
```

Comment:
Can you rename any local vars that have used NS - otherwise could get confusing.